### ISSUE TITLE
Permissions for the group are still present even when the group has been removed from the system

### ISSUE THREAD

**Issue by rabb1t** [haiwen/seafile#1781]
# Environment

Operating system: `CentOS Linux release 7.1.1503 (Core)`
Seafile release: `seafile-pro-server-5.1.11`
Database: `mysql Ver 15.1 Distrib 10.0.27-MariaDB, for Linux (x86_64) using readline 5.1`
# Description

After removing seafile group via web interface the database table `FolderGroupPerm` still holds the record with removed `group_id`
# Steps to reproduce
1. Create a group `group_a`
2. Share folder `folder_a` inside your library `lib_a` to group `group_a` with any permission
3. Set folder permissions for the `folder_a` to group `group_a` with any permission
4. Remove(dismiss) the group `group_a`
5. Set folder permissions for the `folder_a` for the user `user_a` with any permission
6. Check once again the folder permissions for the user `user_a` (they should be gone now)
7. Try to repeat step 5 (and now you should get the error).
# Additional Information

Now to set the folder permission for user on `folder_a` you must remove the `group_id` record from the `seafile.FolderGroupPerm` database table

Also I've attached the django's Internal Server Error when error(on step 7) occures:

```
Internal Server Error: /ajax/repo/f8b8ee79-NNNN-NNNN-NNNN-c0a2ccdf1ae3/get-folder-perm-by-path/
Traceback (most recent call last):
  File "/opt/seafile/seafile-server-latest/seahub/thirdpart/Django-1.8.10-py2.7.egg/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./seahub/auth/decorators.py", line 69, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "./seahub/views/ajax.py", line 1998, in get_folder_perm_by_path
    "group_name": get_group(group_perm.group_id).group_name,
AttributeError: 'NoneType' object has no attribute 'group_name'
```

**Comment by freeplant** [comment]
The fix will be included in the next release.

**Comment by rabb1t** [comment]
Fixed in version `6.0.3 (2016.11.17)`
